extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -197,7 +197,7 @@ export class ResourceTypePickerDialog extends DialogBase {
 				});
 				this._dialogObject.okButton.enabled = messages.length === 0;
 				if (messages.length !== 0) {
-					messages.push(localize('deploymentDialog.VersionInformationDebugHint', "You may find additional details in the debug console."));
+					messages.push(localize('deploymentDialog.VersionInformationDebugHint', "You will need to restart Azure Data Studio if the tools are installed after Azure Data Studio is launched to pickup the updated PATH environment variable. You may find additional details in the debug console."));
```

Comment:
pick up instead of pickup

Also, can we open the debug console via a command link here?

Reply:
thanks, updated the text. right now it is not supported to embed links here. let's do it if we see user requests about this feature.